src/store/flow/block-types/SmartDevices_PhotoResponseBlockStore.ts

```diff
@@ -14,12 +15,23 @@ const actions: ActionTree<IEmptyState, IRootState> = {
     const blankMessageResource = await dispatch('flow/flow_addBlankResourceForEnabledModesAndLangs', null, {root: true})
     props.config = {
       prompt: blankMessageResource.uuid,
+      ...await dispatch('initiateExtraVendorConfig'),
     }
+
+    props.exits = [
+      await dispatch('flow/block_createBlockDefaultExitWith', {
+        props: ({
+          uuid: await (new IdGeneratorUuidV4()).generate(),
+          name: 'Invalid',
```

Comment:
Took the opportunity to update PhotoResponse branch too, to have `Valid/Invalid` exits instead of `1/Default`